### PR TITLE
Databricks benchmark reproduce scripts

### PR DESCRIPTION
**Pull request by eordentlich**

### CODE REVIEW

File: benchmark/databricks/benchmark_utils.sh
Comment (leewyang) [inline benchmark/databricks/benchmark_utils.sh:1]: indentation is a mix of tabs and spaces in this file... should ideally be consistent.

File: benchmark/databricks/init-cpu.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+# set portion of path below after /dbfs/ to dbfs zip file location
+SPARKCUML_ZIP=/dbfs/path/to/spark-rapids-ml.zip
+BENCHMARK_ZIP=/dbfs/path/to/benchmark.zip
+
+# install spark-cuml
+unzip ${SPARKCUML_ZIP} -d /databricks/python3/lib/python3.8/site-packages
```
Comment (leewyang) [inline benchmark/databricks/init-cpu.sh:7]: Should use `SPARK_RAPIDS_ML_ZIP` here (and above).

File: benchmark/databricks/init-pip-cuda-11.8.sh [hunk 2]
```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+# set portion of path below after /dbfs/ to dbfs zip file location
+SPARKCUML_ZIP=/dbfs/path/to/spark-rapids-ml.zip
```
Comment (leewyang) [inline benchmark/databricks/init-pip-cuda-11.8.sh:3]: `SPARK_RAPIDS_ML_ZIP`

File: benchmark/databricks/init-pip-cuda-11.8.sh [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+# install cudf and cuml
+#/databricks/python/bin/pip install cudf-cu11==${RAPIDS_VERSION} cuml-cu11==${RAPIDS_VERSION} --extra-index-url=https://pypi.ngc.nvidia.com
+/databricks/python/bin/pip install cudf-cu11==${RAPIDS_VERSION} \
+cuml-cu11==${RAPIDS_VERSION} \
+pylibraft-cu11==${RAPIDS_VERSION} \
+rmm-cu11==${RAPIDS_VERSION} \
+--extra-index-url=https://pypi.nvidia.com
+
+# install spark-cuml
+unzip ${SPARKCUML_ZIP} -d /databricks/python3/lib/python3.8/site-packages
```
Comment (leewyang) [inline benchmark/databricks/init-pip-cuda-11.8.sh:44]: same

File: benchmark/databricks/run_benchmark.sh [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+    rf_cpu_options=""
+fi
+num_runs=3
+
+sep="=================="
+
+echo
+echo "$sep algo: kmeans $sep"
+for i in `seq $kmeans_runs`; do run_bm kmeans \
+--num_runs 1 \
```
Comment (leewyang) [inline benchmark/databricks/run_benchmark.sh:48]: Could just set `--num-runs $kmeans_runs` in theory (here and elsewhere in fine), unless you need to iterate in bash script explicitly.

**Review by leewyang — COMMENTED** [review]
Some minor notes, otherwise LGTM

File: benchmark/databricks/run_benchmark.sh (hunk 4 above)
Comment (eordentlich) [reply benchmark/databricks/run_benchmark.sh:48]: Things become unstable for some reason with some of the algos, with task failures and longer run times.   We'll have to root cause that.   Could have added the inter-iteration-sleep in the python scripts but went this route.

File: benchmark/databricks/init-pip-cuda-11.8.sh (hunk 3 above)
Comment (eordentlich) [reply benchmark/databricks/init-pip-cuda-11.8.sh]: done all of the above

File: benchmark/databricks/benchmark_utils.sh
Comment (eordentlich) [reply benchmark/databricks/benchmark_utils.sh:1]: weird.  fixed.

**Review by leewyang — APPROVED** [review]

**Pull request merged**